### ISSUE TITLE
Decision review workflow documentation

### ISSUE THREAD

**Issue by pkarman** [department-of-veterans-affairs/caseflow#11852]
It would be helpful to have some illustrated documentation of the Queue workflow process and the various Tasks that are created in that workflow. This would be useful both for Caseflow business understanding and for reporting purposes.

i.e. a picture of the case flow

**Comment by laurjpeterson** [comment]
this [mural](https://app.mural.co/t/workqueue2001/m/workqueue2001/1543852062980/e142cf1394f9a666c6148decfe679de779c695c8) was a start

and this is my start to a more complete version. is this what you mean by illustrated? https://docs.google.com/presentation/d/1Cc84GH7giWHTNxUe3zixH7O-QT77STlptYfud9X8P1Y/edit#slide=id.g3faa773613_0_87

**Comment by pkarman** [comment]
yes! this is exactly what I meant. The google slides look like a great start. Do you feel like they are complete enough to share with the metrics team?

**Comment by pkarman** [comment]
I think we also need some docs at the level of the individual Task model in the code.

I would find the following helpful comments in each `app/models/tasks/*.rb` file:

* what milestone in the overall Caseflow decision review process does this task map to?
* under what conditions would a record of this type be created?
* what parent record types would it potentially have? are any required?
* what child record types would it potentially have? are any required?
* what kind of user would be assigned to this task?
* under what conditions would the status change? from what value to what value?

**Comment by pkarman** [comment]
cc @lomky @lowellrex
